infra/base-images/base-builder/Dockerfile

```diff
@@ -46,7 +46,12 @@ ENV COVERAGE_FLAGS_coverage "-fprofile-instr-generate -fcoverage-mapping -pthrea
 ENV SANITIZER="address"
 ENV FUZZING_ENGINE="libfuzzer"
 
-# Path to fuzzing engine library.
+# Path to fuzzing engine library to support some old users of
```

Comment:
I'd start this comment with "DEPRECATED - DO NOT USE" test followed by the explanation you have